eslint.config.js

```diff
@@ -11,6 +11,10 @@ import yml from "eslint-plugin-yml";
 import { defineConfig } from "eslint/config";
 import tseslint from "typescript-eslint";
 
+const JS_FILES = ["**/*.js", "**/*.mjs"];
+const TS_FILES = ["**/*.ts", "**/*.mts"];
```

Comment:
Neither exist in this repo:

```suggestion
const JS_FILES = ["**/*.js"];
const TS_FILES = ["**/*.ts"];
```

Reply:
Sounds good.  I mainly went with that in the even that some files were added in the future, so they wouldn't fly under the radar.  But, I guess it's unlikely we'll have any added.